authortechnical.Rmd

```diff
@@ -65,12 +69,48 @@ show_template("post-template.Rmd", lang = "yaml",
               yaml_only = TRUE)
 ```
 
-### Add tags
+#### Add subject tags
 
 Add tags to the YAML of your post to make it more findable. Browse [our page that lists all tags in use](https://ropensci.org/tags/) and re-use an existing tag rather than creating a new one e.g. 'packages' exists, so use that, rather than 'package'.
 
 For a post about your peer-reviewed package, use 'Software Peer Review', 'R', 'community', 'packages', the package name, tags that were [topic labels](https://github.com/ropensci/software-review/labels) in your package review such as 'data-access', and any others you see fit.
 
+#### Understanding Twitter cards {#twittercards}
+
+Only read this if you're curious about Twitter cards.
+
+A [Twitter card](https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/abouts-cards) means than when a URL is included in a tweet, what other Twitter users see is not the URL but instead a "card", i.e. the metadata from the URL rendered in a nice preview.
+The Twitter metadata in a post's YAML helps it "look good" when an account like R Weekly Live or other readers link to the post in a tweet. 
+The relevant YAML tags for rOpenSci posts are `title`, `description`, `twitterImg`.
+These same metadata tags might be picked up by other platforms such as Slack.
+
+This is what a tweet about an rOpenSci post looks like by default.
+
+NOTE: code below trying to show tweet screenshot according to https://twitter.com/grrrck/status/1211856156412579840?s=20. Please edit for best way to add tweets here.
+`Error: 'tweet_screenshot' is not an exported object from 'namespace:webshot2'`
+
+NOTE: better to use blog post tweet instead
```

Comment:
Not necessarily because when we tweet the blog posts if there's an image often we'll attach it to the tweet so the Twitter card doesn't appear, and the tweet is already informative.